### PR TITLE
CocoaPods support. Old SDK support

### PR DESCRIPTION
**Pull request by ypopovych**
1. Added CocoaPods Podfile
2. Added GitHub Actions workflows for testing and deploying.
3. Switched it to the older Swift 5.7 and SDKs. I like the StaticBigInt constructor too, but it's supported only in the latest SDK versions :( More practical is `String` constructors for now.
4. Tested on Linux (and it works!)

### CODE REVIEW

**Comment by ypopovych** [comment]
I have one proposal.

I can split this lib into 3 pods. It will allow it to use the same `import` logic as SPM. Workflow scripts though will be crazy.

What do you think? Because right now you need `#if !COCOAPODS` everywhere

**Comment by oscbyspro** [comment]
It sounds better. I prefer CocoaPods as far in the background as possible. In layman's terms, does that mean 3 pod files?

**Comment by ypopovych** [comment]
Yes. It will have 3 Podfiles which will be published on each release. I already added them in the branch: https://github.com/ypopovych/Numberick/tree/pods-swift57

**Comment by ypopovych** [comment]
Closed in favor of: https://github.com/oscbyspro/Numberick/pull/56
